rust/bin-shared/src/tun_device_manager/windows.rs

```diff
@@ -213,14 +212,14 @@ impl Tun {
 
         // SAFETY: we're loading a DLL from disk and it has arbitrary C code in it.
         // The Windows client, in `wintun_install` hashes the DLL at startup, before calling connlib, so it's unlikely for the DLL to be accidentally corrupted by the time we get here.
```

Comment:
If we did everything around WinTUN in here, this safety comment would be a lot easier to verify.